src/nvidia.cc

```diff
@@ -35,33 +35,52 @@
+ * MEM    ${nvidia memfreq [gpu_id]}MHz (${nvidia memfreqmin [gpu_id]}-${nvidia memfreqmax [gpu_id]}MHz)
+ * MTR    ${nvidia mtrfreq [gpu_id]}MHz (${nvidia mtrfreqmin [gpu_id]}-${nvidia mtrfreqmax [gpu_id]}MHz)
+ * PERF   Level ${nvidia perflevel [gpu_id]} (${nvidia perflevelmin [gpu_id]}-${nvidia perflevelmax [gpu_id]}), Mode: ${nvidia perfmode [gpu_id]}
+ * VRAM   ${nvidia memutil [gpu_id]}% (${nvidia memused [gpu_id]}MB/${nvidia memtotal [gpu_id]}MB)
+ * LOAD   GPU ${nvidia gpuutil [gpu_id]}%, RAM ${nvidia membwutil [gpu_id]}%, VIDEO ${nvidia videoutil [gpu_id]}%, PCIe ${nvidia pcieutil [gpu_id]}%
+ * TEMP   GPU ${nvidia gputemp [gpu_id]}°C (${nvidia gputempthreshold [gpu_id]}°C max.), SYS ${nvidia ambienttemp [gpu_id]}°C
+ * FAN    ${nvidia fanspeed [gpu_id]} RPM (${nvidia fanlevel [gpu_id]}%)
+ * 
+ * miscelaneas:
+ * OPENGL ${nvidia imagequality [gpu_id]}
+ * GPU    ${nvidia modelname [gpu_id]} 
  * 
  * --==| NVIDIA Bars |==--
- * LOAD ${nvidiabar gpuutil}
- * VRAM ${nvidiabar memutil}
- * RAM ${nvidiabar membwutil}
- * VIDEO ${nvidiabar videoutil}
- * PCIe ${nvidiabar pcieutil}
- * Fan ${nvidiabar fanlevel}
- * TEMP ${nvidiabar gputemp}
+ * LOAD  ${nvidiabar [height][,width] gpuutil [gpu_id]}
+ * VRAM  ${nvidiabar [height][,width] memutil [gpu_id]}
+ * RAM   ${nvidiabar [height][,width] membwutil [gpu_id]}
+ * VIDEO ${nvidiabar [height][,width] videoutil [gpu_id]}
+ * PCIe  ${nvidiabar [height][,width] pcieutil [gpu_id]}
+ * Fan   ${nvidiabar [height][,width] fanlevel [gpu_id]}
+ * TEMP  ${nvidiabar [height][,width] gputemp [gpu_id]}
+ * 
+ * --==| NVIDIA Gauge |==--
+ * LOAD  ${nvidiagauge [height][,width] gpuutil [gpu_id]}
+ * VRAM  ${nvidiagauge [height][,width] memutil [gpu_id]}
+ * RAM   ${nvidiagauge [height][,width] membwutil [gpu_id]}
+ * VIDEO ${nvidiagauge [height][,width] videoutil [gpu_id]}
+ * PCIe  ${nvidiagauge [height][,width] pcieutil [gpu_id]}
+ * Fan   ${nvidiagauge [height][,width] fanlevel [gpu_id]}
+ * TEMP  ${nvidiagauge [height][,width] gputemp [gpu_id]}
  * 
+ * --==| NVIDIA Graph |==-- (gpu_id is not optional in this case)
+ * LOAD  ${nvidiagraph gpuutil [height][,width] [gradient color 1] [gradient color 2] [scale] [-t] [-l] gpu_id}
+ * VRAM  ${nvidiagraph memutil [height][,width] [gradient color 1] [gradient color 2] [scale] [-t] [-l] gpu_id}
+ * RAM   ${nvidiagraph membwutil [height][,width] [gradient color 1] [gradient color 2] [scale] [-t] [-l] gpu_id}
+ * VIDEO ${nvidiagraph videoutil [height][,width] [gradient color 1] [gradient color 2] [scale] [-t] [-l] gpu_id}
+ * PCIe  ${nvidiagraph pcieutil [height][,width] [gradient color 1] [gradient color 2] [scale] [-t] [-l] gpu_id}
+ * Fan   ${nvidiagraph fanlevel [height][,width] [gradient color 1] [gradient color 2] [scale] [-t] [-l] gpu_id}
+ * TEMP  ${nvidiagraph gputemp [height][,width] [gradient color 1] [gradient color 2] [scale] [-t] [-l] gpu_id}
  */
```

Comment:
Hello Matt,

It doesn't harm to have it there and it is pretty self explanatory.

Let see what the comrades prefer.